isg-503.coffee

```diff
@@ -1,15 +1,28 @@
 deviceTypesCommon = require '@resin.io/device-types/common'
 { networkOptions, commonImg, instructions } = deviceTypesCommon
 
+postProvisioningInstructions = [
+	instructions.BOARD_SHUTDOWN
+	instructions.REMOVE_INSTALL_MEDIA
+	instructions.BOARD_REPOWER
+]
+
 module.exports =
 	version: 1
 	slug: 'isg-503'
-	aliases: [ 'ISG503' ]
 	name: 'ISG 503'
```

Comment:
@RockSupport  Should we change the name to a more descriptive one, perhaps "Rocktech ISG-503" ?
This name will appear in Balena cloud on the image download modal.

Reply:
OK ，Thanks